### ISSUE TITLE
Microcalcifications of the breast

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10084]
**Preferred term label:**
Microcalcifications of the breast

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0520594

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mammary Paget disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000056
- label: Breast microcalcifications
- parents: [Abnormal breast morphology (HP:0031093)](https://hpo.jax.org/app/browse/term/HP:0031093)
- pmid: PMID:30411977
- def: The term breast microcalcification is used to define calcium deposits in the breast with a diameter of less than 1 mm.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10084
- synonyms: Microcalcifications of the breast(NONE)

**Comment by pnrobinson** [comment]
@MickeySegal 

corrected ID to HP:0040403
